changes/2693.bugfix.rst

```diff
@@ -0,0 +1,14 @@
+Implement open() for LoggingStore
+LoggingStore is now a generic class.
+Add _is_open property and setter for WrapperStore
+Use stdout rather than stderr as the default stream for LoggingStore
+Ensure that ZipStore is open before getting or setting any values
+Update equality for LoggingStore and WrapperStore such that the types must be equal.
```

Comment:
```suggestion
Update equality for LoggingStore and WrapperStore such that the type of the store they wrap must be equal for themselves to be equal.
```

Reply:
Your suggestion isn't comprehensive. Previously, this would not raise an error:
`assert LoggingStore(local_store: LocalStore) == local_store: LocalStore`
Now it raises an error because the types are different.

Would `Update equality for LoggingStore and WrapperStore such 'other' must also be a LoggingStore or WrapperStore respectively, rather than only checking the types of the stores they wrap.` be more clear?

Reply:
Ah my mistake - you suggestion sounds good